src/fields/abstractField.js

```diff
@@ -137,8 +137,8 @@ export default {
 					this.schema.onChanged.call(this, this.model, newValue, oldValue, this.schema);
 				}
 
-				if (this.$parent.options && this.$parent.options.validateAfterChanged === true) {
```

Comment:
Not sure why I wrote it like that, must have just been quickly updating something ... it should look like this:

`if (objGet(this.$parent, 'options.validatedAfterChanged', false) === true) { }`

Which is much cleaner looking, to me, than 

`if (this.$parent && this.$parent.options && this.$parent.options.validatedAfterChanged === true) { }`

Reply:
Ok, I get it now. I'm a bit afraid that it will create a dependency on lodash (and raise the barrier to entry to the future contributor) but it is way cleaner, no objections !